src/game/settings.c

```diff
@@ -42,7 +42,10 @@ static int32_t S_ReadUserSettingsATI()
         SetGameScreenSizeIdx(resolution_idx);
     }
 
-    FileRead(&GameSizer, sizeof(double), 1, fp);
+    {
+        float game_sizer;
+        FileRead(&game_sizer, sizeof(double), 1, fp);
+    }
```

Comment:
if this is read to be discarded, possibly Seek over Read?

Reply:
Excellent remark, I'll change it

Reply:
Fixed